docs/csharp/linq/index.md

```diff
@@ -2,54 +2,47 @@
+- Query expressions use many familiar C# language constructs, which make them easy to read.
+- The variables in a query expression are all strongly typed.
+- A query isn't executed until you iterate over the query variable, for example in a `foreach` statement.
+- At compile time, query expressions are converted to standard query operator method calls according to the rules defined in the C# specification. Any query that can be expressed by using query syntax can also be expressed by using method syntax. In some cases, query syntax is more readable and concise. In others, method syntax is more readable. There's no semantic or performance difference between the two different forms. For more information, see [C# language specification](~/_csharpstandard/standard/expressions.md#1220-query-expressions) and [Standard query operators overview](../programming-guide/concepts/linq/standard-query-operators-overview.md).
+- Some query operations, such as <xref:System.Linq.Enumerable.Count%2A> or <xref:System.Linq.Enumerable.Max%2A>, have no equivalent query expression clause and must therefore be expressed as a method call. Method syntax can be combined with query syntax in various ways.
 - Query expressions can be compiled to expression trees or to delegates, depending on the type that the query is applied to. <xref:System.Collections.Generic.IEnumerable%601> queries are compiled to delegates. <xref:System.Linq.IQueryable> and <xref:System.Linq.IQueryable%601> queries are compiled to expression trees. For more information, see [Expression trees](/dotnet/csharp/advanced-topics/expression-trees).
 
 ## How to enable LINQ querying of your data source  
   
 ### In-memory data
 
- There are two ways you can enable LINQ querying of in-memory data. If the data is of a type that implements <xref:System.Collections.Generic.IEnumerable%601>, you can query the data by using LINQ to Objects. If it does not make sense to enable enumeration of your type by implementing the <xref:System.Collections.Generic.IEnumerable%601> interface, you can define LINQ standard query operator methods in that type or create LINQ standard query operator methods that extend the type. Custom implementations of the standard query operators should use deferred execution to return the results.  
+ There are two ways you enable LINQ querying of in-memory data. If the data is of a type that implements <xref:System.Collections.Generic.IEnumerable%601>, you query the data by using LINQ to Objects. If it doesn't make sense to enable enumeration by implementing the <xref:System.Collections.Generic.IEnumerable%601> interface, you define LINQ standard query operator methods, either in that type as [extension methods](../programming-guide/classes-and-structs/extension-methods.md) for that type. Custom implementations of the standard query operators should use deferred execution to return the results.  
```

Comment:
There's something missing in the text following "either in that type"

Reply:
Thanks. It's missing the "or"